packages/vm/lib/runTx.ts

```diff
@@ -68,12 +65,9 @@ export interface RunTxOpts {
   reportAccessList?: boolean
 
   /**
-   * Optional clique Address: if the consensus algorithm is on clique,
-   * and this parameter is provided, use this as the beneficiary of transaction fees
-   * If it is not provided and the consensus algorithm is clique, instead
-   * get it from the block using `cliqueSigner()`
+   * To obtain an accurate tx receipt input the block gas used up until this tx.
    */
-  cliqueBeneficiary?: Address
+  blockGasUsed?: BN
```

Comment:
Ah, yeah, this is a lot more targeted, really somewhat relieved on seeing this change here. 😀